client/asset/eth/eth.go

```diff
@@ -3700,7 +3700,7 @@ func (eth *ETHWallet) monitorBlocks(ctx context.Context) {
 // tipChange callback function is invoked and a goroutine is started to check
 // if any contracts in the findRedemptionQueue are redeemed in the new blocks.
 func (eth *ETHWallet) checkForNewBlocks(ctx context.Context) {
-	ctx, cancel := context.WithTimeout(ctx, 2*time.Second)
```

Comment:
Thirty seconds also seems excessive here... does 8 work? It's only asking for a header.

Reply:
Answered here - https://github.com/decred/dcrdex/pull/3163#discussion_r1935947106 

but the general idea is size isn't the only thing that matters when it comes to network-communications, there can be random delays you can't predict